src/fidesops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -102,6 +116,30 @@ def get_connections(
                 ConnectionConfig.description.ilike(f"%{escape_like(search)}%"),
             )
         )
+
+    if connection_type:
+        query = query.filter(ConnectionConfig.connection_type.in_(connection_type))
+
+    if disabled is not None:
+        query = query.filter(ConnectionConfig.disabled == disabled)
+
+    if test_status:
+        query = query.filter(
+            ConnectionConfig.last_test_succeeded.is_(test_status.str_to_bool())
+        )
+
+    if system_type:
+        if system_type == SystemType.saas:
+            query = query.filter(
+                ConnectionConfig.connection_type == ConnectionType.saas
+            )
+        elif system_type == SystemType.database:
+            query = query.filter(
+                ConnectionConfig.connection_type.notin_(
+                    [ConnectionType.saas, ConnectionType.manual]
```

Comment:
Pointing out that there's really 3 system types right now, saas, database, and manual. I've only added system_type filtering for the two, but I can change.

Reply:
@pattisdr Does that backend support manual connection types right now?

Reply:
yes it does but there's not an easy way to interact with them yet, it is a connection type that requires a user to go manually perform some action and upload data back to fidesops.

Reply:
I see. Maybe we should add it in in the backend just to get ahead of the future update? The frontend can always leave the filtering option out if it isn't supposed to handle it yet.